src/cheri/csv/CHERI_CSR.csv

```diff
@@ -46,3 +46,6 @@ direct write if address didn't change","","✔","","{cheri_default_ext_name}","U
 "stidc","0x580","","S","Read: S, Write: S, <<asr_perm>>","tag=0, otherwise undefined","Update the CSR using <<SCADDR>>.","direct write","","","","{cheri_base_ext_name}","Supervisor thread ID","","","","","","","","","","","","","","","","","","","","",""
 "vstidc","0xA80","","H","Read: VS, Write: VS, <<asr_perm>>","tag=0, otherwise undefined","Update the CSR using <<SCADDR>>.","direct write","","","","{cheri_base_ext_name}","Virtual supervisor thread ID","","","","","","","","","","","","","","","","","","","","",""
 "mtidc","0x780","","M","Read: M, Write: M, <<asr_perm>>","tag=0, otherwise undefined","Update the CSR using <<SCADDR>>.","direct write","","","","{cheri_base_ext_name}","Machine thread ID","","","","","","","","","","","","","","","","","","","","",""
+"mycfg","0x781","","M","RO","Platform specified","","","","","","","Supported CHERI capability encoding","","","","","","","","","","","","","","","","","","","","",""
+"sycfg","0x581","","S","Read: any, Write: (M or VS), <<asr_perm>>","Copies mycfg","WARL","WARL","","","","","","","","","","","","","","","","","","","","","","","","","",""
```

Comment:
+vsycfg?

Reply:
The additions to this file are erroneous, I think, and should be removed.